### ISSUE TITLE
'VNL_Loss' object has no attribute 'cal_VNL_loss'

### ISSUE THREAD

**Issue by LiZhenLiangLee** [YvanYin/VNL_Monocular_Depth_Prediction#27]
Line 193  in VNL_loss.py， `loss = vnl_loss.cal_VNL_loss(pred_depth, gt_depth)` ,  but there isn't `cal_VNL_loss()` in VNL_loss. 
 I guess it should be  `loss = vnl_loss(gt_depth, pred_depth)`  to use the `forward()` ?

**Comment by YvanYin** [comment]
Yes, thanks, I will fix this.
